src/javascript/static/pages/responsible_trading.js

```diff
@@ -0,0 +1,31 @@
+const Scroll = require('../../_common/scroll');
+const Client = require('../../app/base/client');
+const State            = require('../../_common/storage').State;
+const BinarySocket   = require('../../app/base/socket');
```

Comment:
whitespace before `=` should be aligned between these four lines and sorted alphabetically by import path